pkg/apis/config/defaults.go

```diff
@@ -53,26 +53,46 @@ const (
 	// DefaultMaxRevisionContainerConcurrency is the maximum configurable
 	// container concurrency.
 	DefaultMaxRevisionContainerConcurrency int64 = 1000
+
+	// DefaultAllowContainerConcurrencyZero is whether, by default,
+	// containerConcurrency can be set to zero (i.e. unbounded) by users.
+	DefaultAllowContainerConcurrencyZero bool = true
```

Comment:
I am a bit torn on necessity for named constant here, but :shrug:

Reply:
Haha yeah funny you should say that, I spent five or so minutes I’m not getting back umming and ahhing on it too, but in the end it just looks so much nicer with all the constants lined up consistently in defaultConfig I think 🤷‍♂️